README.md

```diff
@@ -48,13 +48,13 @@ many-to-many relationship, we'll need one join table.
 
 Each join table should have, at minimum, **two foreign_key columns**. Each foreign key will represent one of the tables it's joining. In the example of `Doctors` and `Patients`, we would create a **new** join table that has a `doctor_id` column and a `patient_id` column.
 
-We can also add columns as needed to store additional information about the relationship. For example, we may choose to add an `date_of_visit` column which stores a `datetime` value representing when the appointment is, and could be different for each doctor + patient visit.
+We can also add columns as needed to store additional information about the relationship. For example, we may choose to add a `date_of_visit` column which stores a `datetime` value representing when the appointment is, and could be different for each doctor + patient visit.
 
 ## Join Models & Tables
 
-In order to do many-to-many relationships in Rails, convention says to create a new model to represent our join table. The name can technically be anything we want, but really the model name should be as descriptive as possible, and indicate that it represents an *association*.
+In order to do many-to-many relationships in Rails, convention says to create a new model to represent our join table. The name can technically be anything we want, but the model name should be as descriptive as possible, and indicate that it represents an *association*.
 
```

Comment:
no need to change anything in the LP itself, but can mention that they've heard about the concept of joins before in the SQL class. could be an opportunity to ask them "where'd you hear about joins before?" "what were they in SQL?"